homeassistant/components/binary_sensor/insteon_plm.py

```diff
@@ -54,4 +56,6 @@ def device_class(self):
     @property
     def is_on(self):
         """Return the boolean response if the node is on."""
-        return bool(self._insteon_device_state.value)
+        on_val = bool(self._insteon_device_state.value)
+        return (on_val if self._insteon_device_state.name != 'lightSensor' else
```

Comment:
Please convert to a normal if…else if an inline if-statement is spanning two lines.